hazelcast/src/test/java/com/hazelcast/map/SizeEstimatorTest.java

```diff
@@ -48,7 +49,8 @@ public void smoke() throws InterruptedException {
 
     @Test
     public void testSinglePut() throws InterruptedException {
-        long expectedPerEntryHeapCost = 156L;
+        // the expected cost is 8 * reference_cost + (key + record cost)
+        long expectedPerEntryHeapCost = 124 + 8 * JVMUtil.REFERENCE_COST_IN_BYTES;
```

Comment:
Not blocking, but would be nice if we had these figures defined with name, easier to read and wouldn't require comments.